docs/code-organization/naming-convention.mdx

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 2
+---
+
+# Naming Convention
+
+This page contains naming conventions for commit messages and branches.
+
+## Commit Messages
+
+Commit message titles should be in the form `type: Short description of the
+change` or `type(subsystem): Short description of the change` as per
+[Conventional Commits][conventional-commit].
+
+Furthermore, the ticket number (if applicable) should be contained at the
+bottom of the commit message body.
+
+Types may include:
+- `feat`: New feature
+- `fix`: Bug fix of an existing feature
+- `chore`: Changes that are otherwise a chore (bumping a version)
+- `docs`: Documentation changes
+- `refactor`: Change in code for code quality that should not affect functionality
+
+Here is an example of a commit message that should contain more details about
+the changes but is otherwise good:
+
+```
+feat(payment): Add Stripe checkout
+
+This commit adds Stripe integration to the payment page. The STRIPE_API_KEY
+environment variable should be set to the secret key available at Stripe
+dashboard.
+
+[SCSE-24]
+```
```

Comment:
feel free to reword the commit message in the example to contain more details (use positive examples rather than negative examples). 

We can, however, have a 'negative example' of a commit message - one that only contains poor content - in order to highlight multiple problems to the reader.